File: problems/r/railroad2/railroad2.cpp
/*
Author: Rudi César Comiotto Modena
Email: rudi.modena@gmail.com
*/

#include <iostream>

using namespace std;

int main() {
    int x, y;

    cin >> x >> y;

    if (y % 2 == 0) {
        cout << "possible" << endl;
    }
    else {
        cout << "impossible" << endl;
    }
}

File: problems/r/railroad2/railroad2.py
#! /usr/bin/python3
"""
Author: Rudi César Comiotto Modena
Email: rudi.modena@gmail.com
"""

def main():
    x, y = [int(val) for val in input().split()]

    if y % 2 == 0:
        print("possible")
    else:
        print("impossible")


if __name__ == '__main__':
    main()